code/game/objects/items/toys.dm

```diff
@@ -215,6 +215,8 @@
 
 /obj/item/toy/ammo/gun/examine(mob/user)
 	..()
+	if (src.amount_left == 0)
```

Comment:
`if(amount_left <= 0)` is preferable, because it would get a strange case where they somehow went negative. src is not needed.